mifosng-android/src/main/java/com/mifos/api/services/DocumentService.java

```diff
@@ -40,11 +40,11 @@
      */
     @POST("{entityType}/{entityId}/" + APIEndPoint.DOCUMENTS)
     @Multipart
-    Observable<GenericResponse> createDocument(@Path("entityType") String entityType,
-                                               @Path("entityId") int entityId,
-                                               @Part("name") String nameOfDocument,
-                                               @Part("description") String description,
-                                               @Part() MultipartBody.Part typedFile);
+    Observable<DocumentRelatedResponse> createDocument(@Path("entityType") String entityType,
```

Comment:
@tarun0 When we know what will be response for many requests the It's better to follow genericity and Please use `GenericResponse`